README.md

```diff
@@ -7,26 +7,39 @@ A javascript implementation for Heap data structure. Heap base class allows crea
 <img src="https://user-images.githubusercontent.com/6517308/121813242-859a9700-cc6b-11eb-99c0-49e5bb63005b.jpg">
 
 # contents
-* [Install](#install)
-* [require](#require)
-* [import](#import)
-* [API](#api)
-  * [constructor](#constructor)
-  * [insert (push)](#insert-push)
-  * [extractRoot (pop)](#extractroot-pop)
-  * [root (top)](#root-top)
-  * [leaf](#leaf)
-  * [size](#size)
-  * [sort](#sort)
-  * [isValid](#isvalid)
-  * [fix](#fix)
-  * [clone](#clone)
-  * [clear](#clear)
-  * [heapify](#heapify)
-  * [isHeapified](#isheapified)
-  * [Symbol.iterator](#symboliterator)
- * [Build](#build)
- * [License](#license)
+- [@datastructures-js/heap](#datastructures-jsheap)
+- [contents](#contents)
+  - [install](#install)
+  - [require](#require)
+  - [import](#import)
+  - [API](#api)
+    - [constructor](#constructor)
+      - [Heap](#heap)
+        - [TS](#ts)
+        - [JS](#js)
+      - [MinHeap, MaxHeap](#minheap-maxheap)
+        - [TS](#ts-1)
+        - [JS](#js-1)
+    - [insert (push)](#insert-push)
+    - [extractRoot (pop)](#extractroot-pop)
+    - [root (top)](#root-top)
+    - [leaf](#leaf)
+    - [size](#size)
+    - [sort](#sort)
+    - [isValid](#isvalid)
+    - [fix](#fix)
+    - [clone](#clone)
+    - [clear](#clear)
+    - [heapify](#heapify)
+        - [TS](#ts-2)
+        - [JS](#js-2)
+    - [isHeapified](#isheapified)
+      - [TS](#ts-3)
+      - [JS](#js-3)
+    - [Symbol.iterator](#symboliterator)
+    - [toArray](#toarray)
+  - [Build](#build)
+  - [License](#license)
```

Comment:
plz revert the structure changes in readme. It's intentionallly kept without heavy nesting